internal/rbd/rbd_attach.go

```diff
@@ -229,19 +229,19 @@ func createPath(ctx context.Context, volOpt *rbdVolume, cr *util.Credentials) (s
 		mapOptions = append(mapOptions, "--read-only")
 	}
 	// Execute map
-	output, err := execCommand(rbd, mapOptions)
+	stdout, stderr, err := util.ExecCommand(ctx, rbd, mapOptions...)
 	if err != nil {
-		klog.Warningf(util.Log(ctx, "rbd: map error %v, rbd output: %s"), err, string(output))
+		klog.Warningf(util.Log(ctx, "rbd: map error %v, rbd output: %s"), err, stderr)
 		// unmap rbd image if connection timeout
 		if strings.Contains(err.Error(), rbdMapConnectionTimeout) {
 			detErr := detachRBDImageOrDeviceSpec(ctx, imagePath, true, isNbd, volOpt.Encrypted, volOpt.VolID)
 			if detErr != nil {
 				klog.Warningf(util.Log(ctx, "rbd: %s unmap error %v"), imagePath, detErr)
 			}
 		}
-		return "", fmt.Errorf("rbd: map failed %v, rbd output: %s", err, string(output))
+		return "", fmt.Errorf("rbd: map failed %v, rbd output: %s", err, stderr)
```

Comment:
same here?

Reply:
stdout would not contain the error message, so that would not help troubleshooting

Reply:
If thats the case why we have "rbd output: %s" ?

Reply:
the rbd command wrote an error message on stderr, which I consider output? Do you suggest to have the log message `"rbd: map failed with error %v, rbd errror output: %s"` or something? Suggestions welcome.

Reply:
`"rbd: map failed with error %v, rbd errror output: %s"` looks good to me .

Reply:
Ok, updated and rebased. Please check again.